recipes/hyphy/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler:
```

Comment:
Any reason this file and the old compilers are needed?

Reply:
Thanks for looking at this @bgruening

Not sure if this file is still needed, just following what was done in the past. 

The old compilers shouldn't be needed.

@davebx any thoughts?

Reply:
I honestly don't remember adding that. I say remove it and see what happens.